spec/controllers/ops_controller/ops_rbac_spec.rb

```diff
@@ -125,6 +125,35 @@
     end
 
     context "#tenant_set_record_vars" do
+      before do
+        Tenant.seed
+        @tenant = Tenant.root_tenant
+        server = FactoryGirl.create(:miq_server, :zone_id => 1)
```

Comment:
please don't hardcode the zone to something that may not exist, use `:zone => Zone.seed`.

If you are stubbing this, you probably want to just set a default local server:
```
server = EvmSpecHelper.local_miq_server
```